packages/docs-app/src/styles/_examples.scss

```diff
@@ -367,8 +368,16 @@
 }
 
 @keyframes docs-wiggle-rotate {
-  0% { transform: rotate(0); }
-  25% { transform: rotate(-3deg); }
-  75% { transform: rotate(3deg); }
-  100% { transform: rotate(0); }
+  0% {
+    transform: rotate(0);
+  }
+  25% {
+    transform: rotate(-3deg);
+  }
+  75% {
+    transform: rotate(3deg);
+  }
+  100% {
+    transform: rotate(0);
+  }
```

Comment:
prettier is not yet part of the sass toolset. would prefer to make style changes like this all at once when we integrate prettier.